spring-cloud-gcp-dependencies/pom.xml

```diff
@@ -16,7 +16,7 @@
 	<packaging>pom</packaging>
 
 	<properties>
-		<google-cloud-bom.version>0.32.0-alpha</google-cloud-bom.version>
+		<google-cloud-bom.version>0.32.1-alpha-SNAPSHOT</google-cloud-bom.version>
```

Comment:
This is pending a new google-cloud-java release.
It will have to be change once the release is made.
This PR can't be merged until the new release is available.

Reply:
Therefore this PR can't be review until the finalized target API.
Or we really will have a spare time...

Reply:
I don't expect the API to change at all, but you can wait until we have an actual release if you prefer!